operator/internal/handlers/internal/openshift/alertmanager.go

```diff
@@ -28,3 +28,20 @@ func AlertManagerSVCExists(ctx context.Context, stack lokiv1.LokiStackSpec, k k8
 
 	return err == nil, nil
 }
+
+// UserWorkloadAlertManagerSVCExists returns true if the Openshift User Workload AlertManager is present in the cluster.
+func UserWorkloadAlertManagerSVCExists(ctx context.Context, stack lokiv1.LokiStackSpec, k k8s.Client) (bool, error) {
+	if stack.Tenants == nil || (stack.Tenants.Mode != lokiv1.OpenshiftLogging && stack.Tenants.Mode != lokiv1.OpenshiftNetwork) {
+		return false, nil
+	}
```

Comment:
The tenant mode `openshift-network` does not support the application tenant. In fact they have a single cluster-admin-scoped tenant called `network` (See https://github.com/grafana/loki/blob/main/operator/internal/manifests/openshift/configure.go#L24)

Reply:
I didn't know that